src/ci/build.sh

```diff
@@ -21,7 +21,7 @@ COMMON_SCRIPT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/common.sh"
 source "${COMMON_SCRIPT}"
 
 # all the prep is done, lets run the build!
-MVN_CMD="mvn -s src/ci/settings.xml"
+MVN_CMD="mvn -s src/ci/settings.xml -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn"
```

Comment:
You can also use `mvn -q` to "quiet" Maven.

Reply:
Yeah, I want some of the output so i can debug an odd build, i just wish there was a `--verbose 2` or something flag